common/styles/_phone-input.scss

```
@@ -0,0 +1,445 @@
+      animation: ds-pi-fadein 200ms cubic-bezier(0.4, 0, 0.2, 1) both;
+    }
+  }
+
+  #{$input-a11y-root} > .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)